### PR TITLE
move arguments of request wrapper methods in jac and fetcher into opts

### PR DESCRIPTION
**Pull request by ArtSuslov**

### CODE REVIEW

File: packages/jac/src/types/json-api.ts [hunk 1]
```diff
@@ -61,7 +61,11 @@ export type JsonApiResponseLinks = {
...
 }
 
-export type JsonApiClientRequestOpts = FetcherRequestConfig & {
+export type JsonApiClientRequestOpts = Omit<
+  JsonApiClientRequestConfig,
+  'query'
+> & {
+  query?: JsonApiClientRequestQuery
   isNeedRaw?: boolean
 }
```
Comment (napalmpapalam) [inline packages/jac/src/types/json-api.ts:70]: no need to omit, u could just overwrite it with `&`

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0-rc.8] - 2023-04-29
+### Changed
+- `@distributedlab/fetcher` - Moved `query` and `body` arguments of `request` wrapping methods into `opts`
```
Comment (napalmpapalam) [inline CHANGELOG.md:9]: U haven't changed examples

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

File: packages/jac/src/types/json-api.ts (hunk 1 above)
Comment (ArtSuslov) [reply packages/jac/src/types/json-api.ts:70]: discussed

File: CHANGELOG.md (hunk 2 above)
Comment (ArtSuslov) [reply CHANGELOG.md:9]: fixed

**Review by napalmpapalam — APPROVED** [review]

**Pull request merged**